reactor-netty-core/src/main/java/reactor/netty/resources/PooledConnectionProvider.java

```diff
@@ -132,6 +136,13 @@ public final Mono<? extends Connection> acquire(
 					log.debug("Creating a new [{}] client pool [{}] for [{}]", name, poolFactory, remoteAddress);
 				}
 
+				if (expectedConnectionPools > Builder.EXPECTED_CONNECTION_POOLS_DISABLED && connectionPoolCount.incrementAndGet() > expectedConnectionPools) {
+					if (log.isWarnEnabled()) {
+						log.warn("Connection pool creation limit exceeded: {} pools created, maximum expected is {}", connectionPoolCount.get(),
+								expectedConnectionPools);
```

Comment:
IMO it is better to call it maximum connection pools. Wdyt?

```suggestion
				if (maxConnectionPools > Builder.MAX_CONNECTION_POOLS && connectionPoolCount.incrementAndGet() > maxConnectionPools) {
					if (log.isWarnEnabled()) {
						log.warn("Connection pool creation limit exceeded: {} pools created, maximum expected is {}", connectionPoolCount.get(),
								maxConnectionPools);
```